File: Common/InMotionGIT.Common/Services/Contracts/GeoInformation.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract>
    Public Class GeoInformation

        <DataMember(Name:="as")>
        Public Property [As]() As String
            Get
                Return m_As
            End Get
            Set
                m_As = Value
            End Set
        End Property

        Private m_As As String

        <DataMember(Name:="city")>
        Public Property City() As String
            Get
                Return m_City
            End Get
            Set
                m_City = Value
            End Set
        End Property

        Private m_City As String

        <DataMember(Name:="country")>
        Public Property Country() As String
            Get
                Return m_Country
            End Get
            Set
                m_Country = Value
            End Set
        End Property

        Private m_Country As String

        <DataMember(Name:="countryCode")>
        Public Property CountryCode() As String
            Get
                Return m_CountryCode
            End Get
            Set
                m_CountryCode = Value
            End Set
        End Property

        Private m_CountryCode As String

        <DataMember(Name:="isp")>
        Public Property Isp() As String
            Get
                Return m_Isp
            End Get
            Set
                m_Isp = Value
            End Set
        End Property

        Private m_Isp As String

        <DataMember(Name:="lat")>
        Public Property Lat() As Double
            Get
                Return m_Lat
            End Get
            Set
                m_Lat = Value
            End Set
        End Property

        Private m_Lat As Double

        <DataMember(Name:="lon")>
        Public Property Lon() As Double
            Get
                Return m_Lon
            End Get
            Set
                m_Lon = Value
            End Set
        End Property

        Private m_Lon As Double

        <DataMember(Name:="org")>
        Public Property Org() As String
            Get
                Return m_Org
            End Get
            Set
                m_Org = Value
            End Set
        End Property

        Private m_Org As String

        <DataMember(Name:="query")>
        Public Property Ip() As String
            Get
                Return m_Ip
            End Get
            Set
                m_Ip = Value
            End Set
        End Property

        Private m_Ip As String

        <DataMember(Name:="region")>
        Public Property Region() As String
            Get
                Return m_Region
            End Get
            Set
                m_Region = Value
            End Set
        End Property

        Private m_Region As String

        <DataMember(Name:="regionName")>
        Public Property RegionName() As String
            Get
                Return m_RegionName
            End Get
            Set
                m_RegionName = Value
            End Set
        End Property

        Private m_RegionName As String

        <DataMember(Name:="status")>
        Public Property Status() As String
            Get
                Return m_Status
            End Get
            Set
                m_Status = Value
            End Set
        End Property

        Private m_Status As String

        <DataMember(Name:="timezone")>
        Public Property Timezone() As String
            Get
                Return m_Timezone
            End Get
            Set
                m_Timezone = Value
            End Set
        End Property

        Private m_Timezone As String

        <DataMember(Name:="zip")>
        Public Property Zip() As String
            Get
                Return m_Zip
            End Get
            Set
                m_Zip = Value
            End Set
        End Property

        Private m_Zip As String
    End Class

End Namespace

File: Common/InMotionGIT.Common.Domain/DataAcceess/GeoInformation.cs
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Domain.DataAccess;


[DataContract]
public class GeoInformation
{

    [DataMember(Name = "as")]
    public string As
    {
        get
        {
            return m_As;
        }
        set
        {
            m_As = value;
        }
    }

    private string m_As;

    [DataMember(Name = "city")]
    public string City
    {
        get
        {
            return m_City;
        }
        set
        {
            m_City = value;
        }
    }

    private string m_City;

    [DataMember(Name = "country")]
    public string Country
    {
        get
        {
            return m_Country;
        }
        set
        {
            m_Country = value;
        }
    }

    private string m_Country;

    [DataMember(Name = "countryCode")]
    public string CountryCode
    {
        get
        {
            return m_CountryCode;
        }
        set
        {
            m_CountryCode = value;
        }
    }

    private string m_CountryCode;

    [DataMember(Name = "isp")]
    public string Isp
    {
        get
        {
            return m_Isp;
        }
        set
        {
            m_Isp = value;
        }
    }

    private string m_Isp;

    [DataMember(Name = "lat")]
    public double Lat
    {
        get
        {
            return m_Lat;
        }
        set
        {
            m_Lat = value;
        }
    }

    private double m_Lat;

    [DataMember(Name = "lon")]
    public double Lon
    {
        get
        {
            return m_Lon;
        }
        set
        {
            m_Lon = value;
        }
    }

    private double m_Lon;

    [DataMember(Name = "org")]
    public string Org
    {
        get
        {
            return m_Org;
        }
        set
        {
            m_Org = value;
        }
    }

    private string m_Org;

    [DataMember(Name = "query")]
    public string Ip
    {
        get
        {
            return m_Ip;
        }
        set
        {
            m_Ip = value;
        }
    }

    private string m_Ip;

    [DataMember(Name = "region")]
    public string Region
    {
        get
        {
            return m_Region;
        }
        set
        {
            m_Region = value;
        }
    }

    private string m_Region;

    [DataMember(Name = "regionName")]
    public string RegionName
    {
        get
        {
            return m_RegionName;
        }
        set
        {
            m_RegionName = value;
        }
    }

    private string m_RegionName;

    [DataMember(Name = "status")]
    public string Status
    {
        get
        {
            return m_Status;
        }
        set
        {
            m_Status = value;
        }
    }

    private string m_Status;

    [DataMember(Name = "timezone")]
    public string Timezone
    {
        get
        {
            return m_Timezone;
        }
        set
        {
            m_Timezone = value;
        }
    }

    private string m_Timezone;

    [DataMember(Name = "zip")]
    public string Zip
    {
        get
        {
            return m_Zip;
        }
        set
        {
            m_Zip = value;
        }
    }

    private string m_Zip;
}